static/js/caseNotes.js

```diff
@@ -0,0 +1,32 @@
+$(function() {
+  var typeSelect = $('#type')
+  var subTypeSelect = $('#subType')
+  var subTypesDiv = $('#subTypes')
+  var subTypesUrl = $('#subTypesUrl').val()
+
+  typeSelect.on('change', function(e) {
+    var selectedTypeOption =
+      e.target && e.target.options && e.target.options && e.target.options[e.target.options.selectedIndex]
```

Comment:
is that a repetition of `e.target.options` or are my eyes blurring

Reply:
Ah, good spot!